api/GovStack_Registration_BB_API.json

```diff
@@ -435,156 +430,65 @@
 			}
 		},
 
-		"/service/operatorFileList": {
```

Comment:
both operatorFileList and applicantFileList are actually doing the same (maybe with different access permissions). Similar to /tasks I suggest to just have a single endpoint getting existing applications (--> `/applications`)

Reply:
I like the idea. You are free to optimize.